src/puppetlabs/puppetdb/cli/export.clj

```diff
@@ -24,20 +26,39 @@
             [slingshot.slingshot :refer [throw+ try+]])
   (:import [java.net URL]))
 
-(def ^:private api-version :v4)
+(def admin-api-version :v1)
+(def query-api-version :v4)
+(def ^:private command-versions
+  ;; This is not ideal that we are hard-coding the command version here, but
+  ;;  in our current architecture I don't believe there is any way to introspect
+  ;;  on which version of the `replace catalog` matches up with the current
+  ;;  version of the `catalog` endpoint... or even to query what the latest
+  ;;  version of a command is.  We should improve that.
+  {:replace_catalog 6
+   :store_report 5
+   :replace_facts 4})
```

Comment:
This is different from what we've done before, I'd probably want to leave that for a separate ticket.